### PR TITLE
Update FileLogTraceListener.xml

### PR DESCRIPTION
**Pull request by KathleenDollard**
Please check that this is the right place to make the change, and let me know if it should be changed in source instead.

## Summary

Describe your changes here.

Fixes: https://github.com/dotnet/vblang/issues/294

### CODE REVIEW

**Comment by MohammadHamdyGhanem** [comment]
I suggest to add this note, and you can rephrase it as you want:
> Note that Append(False) erases the file data when it is called for the first time only. after that it appends the new data to the end of the file as long as the current instance of the FileLogTraceListener is append. If the Listener is closed and then reopened, calling Append(False) again will erase the file data.

File: xml/Microsoft.VisualBasic.Logging/FileLogTraceListener.xml [hunk 1]
```diff
@@ -227,14 +227,14 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Determines whether to append the output to the current file or write it to a new file.</summary>
+        <summary>Determines whether to append the output to the current file or write it to a new or existing file.</summary>
         <value>
           <see langword="Boolean" />, with <see langword="True" /> indicating that the output is appended to the current file, and <see langword="False" /> indicating that output is written to a new file. The default setting for this property is <see langword="True" />.</value>
```
Comment (rpetrusha) [inline xml/Microsoft.VisualBasic.Logging/FileLogTraceListener.xml]: The \<value> section needs to be changed as well. It's also awkward (the Boolean is repeated from the line above). Better would be:

> \<see langword="True" /> to append the output to the current file, and \<see langword="False" /> to write the output to a new or existing file. The default setting...

Comment (KathleenDollard) [reply xml/Microsoft.VisualBasic.Logging/FileLogTraceListener.xml]: Maybe: 

> <see langword="True" /> to append the output to the current file.  If the file does not exist, <see langword="False" /> writes output to a new file. If the file already exists, <see langword="False" /> deletes the previous contents of the file before writing the output. The default setting..

**Comment by MohammadHamdyGhanem** [comment]
@KathleenDollard
I made more modifications:
> Note: "Append = False" affects the log file once in the life time of the FileLogTraceListener instance. This means the FileLogTraceListener will delete existing contents of the log file only before it writes the first log, then FileLogTraceListener will append every new trace data to the end of the file. Be aware that If the Listener is closed and then reopened while Append = False, it will erase the log file data again before writing new logs! So, be careful if you set the value of the Append property to be "false" in a configuration file!

**Comment by KathleenDollard** [comment]
@MohammadHamdyGhanem We will get this!

My changes are to improve readability and better align to our standards. @rpetrusha can weigh in as well, including whether Ron thinks a note is the right way to express this suggestion to help others avoid the problem you encountered. 

> "Note: `Append = False` affects the log file during the life time of the FileLogTraceListener instance. This means the FileLogTraceListener will delete existing contents of the log file once after the value is set or the file is opened and before it writes the first log.  After that FileLogTraceListener will append additional trace data to the end of the file. Be aware that If the FileLogTraceListener instance is closed and then reopened while Append is set to False, it will erase the log file data before writing additional logs. This can lead to missing data, particularly if you set the value of the Append property to "false" in a configuration file.

**Comment by MohammadHamdyGhanem** [comment]
@KathleenDollard
After s second thought, I think we can achive the two modifications in one:

Append = true: 
The log file will be opened in "Append Mode", to keep existing logs. Current writing position will be at end of the file, so that any new logs are appended to the end of the file.
Append = false: 
The log file will be append in "Truncate Mode", where all data is deleting when the log file is opened.

See File.Open Method for more info about file modes.

**Comment by mairaw** [comment]
@rpetrusha do you want to help finish this PR so we can merge it?

**Comment by rpetrusha** [comment]
I should be able to resolve the issues once 4.8 is finished, @mairaw.

**Pull request merged**